selfdrive/timezoned.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import time
+import json
+import subprocess
+from common.params import Params
+from selfdrive.swaglog import cloudlog
+
+from selfdrive.hardware import TICI
+
+if TICI:
```

Comment:
I think as much as possible should be not EON, we could just get this in manager for PC

Reply:
The problem is that the import will fail on NEOS. Until we ship a new NEOS I'd like to gate the import.